cmd/serverless/main.go

```diff
@@ -391,6 +395,17 @@ func runAgent(stopCh chan struct{}) (daemon *serverless.Daemon, err error) {
 	return
 }
 
+func buildGlobalTagsMap(functionARN string, functionName string, region string, awsAccountID string) map[string]string {
```

Comment:
nit: There is a function called 'aws.GetARNTags()' that retrieves functionName,functionArn, region,  awsAccount and account_id tags. Might be worth reusing.